### PR TITLE
Fix docblocks to comments

### PR DESCRIPTION
**Pull request by keradus**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | ? |
| Fixed tickets | N/A |
| License | MIT |
| Doc PR | N/A |

Change docblock into comment when it's not a proper docblock.

### CODE REVIEW

**Comment by iltar** [comment]
You're actually killing the docblocks, what's the point of this?

**Comment by keradus** [comment]
Yes, I change the docblock into comment where it is not a docblock in a meaning of PHPDoc.

Part of regression of CS Fixer to make fabbot.io happy when it update to 1.5.

**Comment by jaytaph** [comment]
There is a subtle change between docblocks and comments. (for one, being tokenized as T_DOCBLOCK instead of T_COMMENT in PHP). In these cases they should be considered comments, and not docblocks. 

But, if this change is going to happen, it should be done in every single file through symfony2 (also, i'm not sure if there is anything for this provided in the coding standards?)

**Comment by keradus** [comment]
@jaytaph yeah, it was done for whole 2.3 branch.

**Comment by iltar** [comment]
@keradus I think that as long as `@see` or any other annotations, whether it be build in or custom, should be treated as doc blocks.

File: src/Symfony/Component/Serializer/Encoder/XmlEncoder.php [hunk 1]
```diff
@@ -313,11 +313,11 @@ private function buildXml(\DOMElement $parentNode, $data, $xmlRootNodeName = nul
                 } elseif ($key === '#') {
                     $append = $this->selectNodeType($parentNode, $data);
                 } elseif (is_array($data) && false === is_numeric($key)) {
-                    /**
+                    /*
                      * Is this array fully numeric keys?
```
Comment (iltar) [inline src/Symfony/Component/Serializer/Encoder/XmlEncoder.php]: I think this comment is obsolete as it just tells _what_ the line below does, not _why_ it does it. Maybe it should be removed instead?

File: src/Symfony/Component/Validator/Constraints/CardSchemeValidator.php [hunk 2]
```diff
@@ -56,39 +56,39 @@ class CardSchemeValidator extends ConstraintValidator
...
          * All MasterCard numbers start with the numbers 51 through 55. All have 16 digits.
          */
         'MASTERCARD' => array(
             '/^5[1-5][0-9]{14}$/',
         ),
-        /**
+        /*
          * All Visa card numbers start with a 4. New cards have 16 digits. Old cards have 13.
          */
         'VISA' => array(
```
Comment (iltar) [inline src/Symfony/Component/Validator/Constraints/CardSchemeValidator.php]: To be honest, I think that 3 lines of comment for 1 or 2 lines of description is overkill. In my opinion the `// single line comment` would be sufficient.

**Comment by keradus** [comment]
> @keradus I think that as long as @see or any other annotations, whether it be build in or custom, should be treated as doc blocks.

IMO here there are simple comments, and comments can have annotations too

**Comment by iltar** [comment]
@keradus except that parsers don't check the comments, but do the check the docblock, there's a difference between the token wise. as @jaytaph just stated:
> There is a subtle change between docblocks and comments. (for one, being tokenized as T_DOCBLOCK instead of T_COMMENT in PHP)

Parsers such as doctrine only take T_DOCBLOCK.

File: src/Symfony/Component/Serializer/Encoder/XmlEncoder.php (hunk 1 above)
Comment (keradus) [inline src/Symfony/Component/Serializer/Encoder/XmlEncoder.php]: Maybe. Not sure. Just change type of doc/comment here.

**Comment by keradus** [comment]
Yes, but I think it is not a case here.

**Comment by fabpot** [comment]
Thank you @keradus.

File: src/Symfony/Component/HttpFoundation/Session/Storage/Handler/NativeSessionHandler.php [hunk 3]
```diff
@@ -11,11 +11,8 @@
 
 namespace Symfony\Component\HttpFoundation\Session\Storage\Handler;
 
-/**
- * Adds SessionHandler functionality if available.
- *
- * @see http://php.net/sessionhandler
- */
+// Adds SessionHandler functionality if available.
+// @see http://php.net/sessionhandler
```
Comment (GrahamCampbell) [inline src/Symfony/Component/HttpFoundation/Session/Storage/Handler/NativeSessionHandler.php]: You shouldn't be using an annotation outside of a phpdoc style comment.

**Pull request closed without merging**